client/src/Components/pages/Official.js

```diff
@@ -15,14 +17,23 @@ const Official = () => {
     const result = await allOfficialAccountOfUser()
     if (result) {
       setOfficialAccountList(result)
+      setFilterOAList(result)
     }
   }
 
+  const handleSearchChange = (value) => {
+    let filter = officialAccountsList.filter((officialAccount) => {
+      return officialAccount.accountName.toLowerCase().includes(value.toLowerCase().trim()) ||
+             officialAccount.description.toLowerCase().includes(value.toLowerCase().trim())
+    })
+    setFilterOAList(filter)
+  }
+
   return (
     <div className='official'>
       <Container>
-        <SearchEngine></SearchEngine>
-        <TableBeauty result={officialAccountsList}></TableBeauty>
+        <SearchEngine handleSearchChange={handleSearchChange} result={filterOAList}></SearchEngine>
+        <TableBeauty result={filterOAList}></TableBeauty>
```

Comment:
Update code to beautiful :smile: 
```suggestion
        <SearchEngine handleSearchChange={handleSearchChange} />
        <TableBeauty result={filterOAList} />
```